pr_consistency/1.get_merged_prs.py

```diff
@@ -31,8 +35,9 @@ def parse_isoformat(string):
 
 
 # Get handle to repository
-g = Github(*get_credentials())
-repo = g.get_repo(REPOSITORY)
+g = GitHub(*get_credentials())  # NOTE: With authentication
+# g = GitHub()  # NOTE: No authentication (has API limit)
```

Comment:
Note: I added option for anon access (albeit disabled) in case you don't think API limit is a problem in the future.